getting-started/01.TEMPLATES.md

```diff
@@ -1,14 +1,14 @@
-# Templates
+# Prompt Template
 
 1. [Prompts](./00.PROMPTS.md)
-2. [**Templates**](./01.TEMPLATES.md)
-3. [Actions](./02.ACTIONS.md)
-4. [Plans](./03.PLANS.md)
-5. [Chaining](./04.CHAINING.md)
-6. [Storage](./05.STORAGE.md)
-7. [Turns](./06.TURNS.md)
-
-A simple and powerful way to
+2. [**Prompt Template**](./01.TEMPLATES.md)
```

Comment:
Did you want to rename this file to prompt templates? (optional)

Reply:
done